_docs/tasks/basic-access-control.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Enabling Simple Access Control
+overview: This task shows how to use Istio to control access to a service.
+          
+order: 41
+
+layout: docs
+type: markdown
+---
+
+This task shows how to use Istio to control access to a service.
+
+## Before you begin
+
+* Setup Istio by following the instructions in the
+  [Installation guide](/docs/tasks/installing-istio.html).
+
+* Deploy the [bookinfo](/docs/samples/bookinfo.html) sample application.
```

Comment:
/docs/samples/bookinfo.html is consistent with all other tasks here. do we need a change PR for all those ?